wikimedia/wikipedia-ios
# Add edit tags batch 3

**Phabricator:** https://phabricator.wikimedia.org/T366732

### Notes
This PR adds our final batch of edit tags. It adds two tags to image recommendations edits - `App suggested edit` and `App image add top`.

### Test Steps
1. Add an image recommendation (be sure developer settings bypass post edit toggle is OFF).
2. Check the edit on Desktop article history. Confirm `App suggested edit` and `App image add top` are added as edit tags.

[Example](https://en.wikipedia.org/w/index.php?title=Syrnolinae&action=history) from developer testing
